portfolio/src/main/webapp/community.js

```diff
@@ -0,0 +1,57 @@
+function renderComments() {
+let quantity = document.getElementById("quantity").value;
+  fetch('/comments?quantity=' + quantity).then(response => response.json()).then((conversation) => {
+    
+    let commentSection = document.getElementById('submitted-comments');
+    while (commentSection.lastChild) {
+        commentSection.removeChild(commentSection.lastChild);
+    }
+
+    let commentCount = 0;
+    conversation.forEach((comment) => {
+        createComment(comment, commentCount%2);
+        commentCount++;
+    });
+  });
+}
+
+function createComment(comment, color) {
+    const commentSection = document.getElementById('submitted-comments');
+    const commentDivElement = document.createElement('div');
+    const usernameElement = document.createElement('p');
+    const commentElement = document.createElement('p');
+    const exitButtonElement = document.createElement('button');
+
+    let commentDivClasses = new Array(2);
+    commentDivClasses[0] = 'comment-div';
+
+    exitButtonElement.className = 'left';
+
+    if (color===0) {
+        commentDivClasses[1] = "lavender";
+    } else {
+        commentDivClasses[1] = "lavender-blush";
+    }
+
+    console.log(commentDivClasses);
```

Comment:
Remove the console.log call before submitting =)

Reply:
Removed